examples/kdbget.c

```diff
@@ -8,12 +9,52 @@ int main()
 	KDB *handle = kdbOpen(key);
 
 	kdbGet(handle, myConfig, key);
-	// TODO: check for errors and warnings in key
 
-	keyDel(key);
+	//check for error
+	if(keyGetMeta (key,"error"))
```

Comment:
actually it would be better to check the return value of kdbGet() and do error handling based on that